src/public/modules/forms/base/components/feedback-form.client.component.js

```diff
@@ -31,19 +33,14 @@ function feedbackController(FormFeedback, Toastr) {
         isPreview: vm.isPreview,
       }
 
-      FormFeedback.postFeedback(
-        {
-          formId: vm.formId,
-        },
-        feedback,
-      ).then(
+      FormFeedback.postFeedback(vm.formId, feedback).then(
         function (_response) {
           vm.isSubmitted = true
           vm.isLoading = false
           Toastr.success('Thank you for your submission!')
+          $scope.$apply()
```

Comment:
if we use `$scope.$apply()` here, do we need to do so in the error handling as well? I'm not sure - if that's the case then maybe we should just stick to `$q.when`?

Reply:
thanks, changed to $q.when